content/en/docs/setup/install/operator/index.md

```diff
@@ -138,8 +138,8 @@ spec:
 EOF
 {{< /text >}}
 
-You can also enable or disable components and modify resource settings.
-For example, to enable the `Grafana` component and increase pilot memory requests:
```

Comment:
Rather than removing the part about enable/disable components, why not change the example to egress-gateway instead?

Reply:
Also, shouln't `pilot` -> `istiod`?

Reply:
+1

thanks for doing this!

Reply:
> Rather than removing the part about enable/disable components, why not change the example to egress-gateway instead?

Done

> Also, shouln't pilot -> istiod?

The [default](https://github.com/istio/istio/blob/master/manifests/profiles/default.yaml#L22) and [other](https://github.com/istio/istio/blob/master/manifests/profiles/demo.yaml#L47) profiles still use `pilot`.

Reply:
> Rather than removing the part about enable/disable components, why not change the example to egress-gateway instead?

Done

> Also, shouln't pilot -> istiod?

The [default](https://github.com/istio/istio/blob/master/manifests/profiles/default.yaml#L22) and [other](https://github.com/istio/istio/blob/master/manifests/profiles/demo.yaml#L47) profiles still use `pilot`.